play/android-publisher/src/main/kotlin/com/github/triplet/gradle/androidpublisher/internal/TrackManager.kt

```diff
@@ -71,8 +71,6 @@ internal class DefaultTrackManager(
             createDefaultTrack(config)
         }
 
-        track.maybeCopyChangelogFromPreviousRelease(config.trackName)
```

Comment:
Dumb question cuz I don't remember how anything works, but why was this removed?

Reply:
Instead of calling `Track.maybeCopyChangelogFromPreviousRelease` we now call `TrackRelease.maybeCopyChangelogFromPreviousRelease` in places where we create a `TrackRelease`. So this functionality is still there, just in different places.